internal/namespaces/config/commands.go

```diff
@@ -29,10 +31,50 @@ func GetCommands() *core.Commands {
 }
 
 func configRoot() *core.Command {
+	configPath := scw.GetConfigPath()
+
 	return &core.Command{
-		Short:     `Config file management`,
-		Long:      `Manage your Scaleway CLI config file.`,
+		Short: `Config file management`,
+		Long: `Config management engine is common across all Scaleway developer tools (CLI, terraform, SDK,... ). It allows to handle your Scaleway config through two ways: environment variables or/and config file.
+
+Scaleway config file is self-documented, we recommend you to have a look at it at least once before using Scaleway developer tools: ` + terminal.Style(configPath, color.Bold, color.FgBlue) + `
+
+In this CLI, ` + terminal.Style(`config file attributes are applied prior to the environment variables`, color.Bold) + `: environment variable > config attribute.
```

Comment:
What do you think about `In this CLI, environment variables have priority over the configuration file.`?